src/uu/ptx/src/ptx.rs

```diff
@@ -88,6 +88,17 @@ fn read_word_filter_file(
     Ok(words)
 }
 
+fn read_char_filter_file(
```

Comment:
Could you please document the function? thanks

Reply:
done, I hope okay like that.

Reply:
yeah, thanks